spec/jobs/distribution_reminder_mailer_job_spec.rb

```diff
@@ -34,35 +13,23 @@
     let(:distribution_with_reminder) { create(:distribution, partner: partner_with_reminders) }
 
     it "does not send mail for past distributions" do
-      Sidekiq::Testing.inline! do
-        expect do
-          DistributionReminderJob.perform_async(past_distribution.id)
-        end.to_not change { ActionMailer::Base.deliveries.count }
-      end
+      DistributionReminderJob.perform_now(past_distribution.id)
+      expect(DistributionMailer.method(:reminder_email)).not_to be_delayed(past_distribution)
```

Comment:
Many changes like this -- instead of checking how many jobs are queued, we look at what their results should be.